data/human/human missions.txt

```diff
@@ -4353,9 +4353,10 @@ mission "Spaceport Reminder Setter"
 		"spaceport reminder month" = "month"
 		fail
 
-# Use of "AA" at the start of the mission name is to make it offer only if no other missions have.
-mission "AA Spaceport Reminder Resetter"
+# minor with order -1 means it is offered only if no other missions apply.
+mission "Spaceport Reminder Resetter"
 	minor
+	order -1
```

Comment:
The way that minor missions are currently coded to be chosen actually means that the last minor mission in the list is the one that gets run, so we actually want this mission to be higher in the list.
```suggestion
# The minor mission with the lowest precedent is what gets offered, so use
# an offer order of 1 so that this mission is discarded first if there are other
# minor missions available.
mission "Spaceport Reminder Resetter"
	minor
	order 1
```

Reply:
If that's the case then it sounds like your SortHelper is flipped. Higher order should mean that the mission is closer to the front of the list, not the back.

Reply:
We don't expect content creators to know SQL, nor do we seek to model ourselves after it.

Reply:
Then perhaps we rename the token to "precedence".